### PR TITLE
Test reduced capabilities (SOFTWARE-4477)

### PR DESCRIPTION
**Pull request by brianhlin**
Unsurprisingly, this works and now we can test it!

EDIT: Reading over the ticket, it sounds like we can try reduced caps for the `cvmfsexec` case now that we moved over to the GlideinWMS Singularity scripts in #28

### CODE REVIEW

**Comment by matyasselmeci** [comment]
Dunno about "unsurprisingly" but hey, not complaining. So did you check that the the two CVMFS volumes got mounted (i.e. exist and are non-empty)?

**Comment by brianhlin** [comment]
@matyasselmeci looks like the `cvmfsexec` test cases failed. The reduced caps on the bind-mounted CVMFS look, good though: by checking the value of `HAS_SINGULARITY`, we're also testing CVMFS availability within the container.

**Comment by brianhlin** [comment]
@DrDaveD we added some CI tests to verify some bits of the backfill container and we're trying to test the following capabilities: https://github.com/opensciencegrid/osgvo-docker-pilot/pull/35/files#diff-3ce2aeee6e5389638907be79b09253b67574fb2151dab8ce4d2f68d2852d5cf1R8-R10. These work fine for the bindmount case but don't for `cmvfsexec`. Do you have thoughts on what capabilities would work now that we've moved the containers over to the GlideinWMS Singularity scripts?

**Comment by DrDaveD** [comment]
I think you're just missing `--device /dev/fuse` which is also recommended in the [install-singularity](https://opensciencegrid.org/docs/worker-node/install-singularity/#enabling-unprivileged-singularity) documentation.

**Comment by brianhlin** [comment]
@matyasselmeci @rynge tests look happy now!

File: tests/test_inside_gha.sh [hunk 1]
```diff
@@ -5,6 +5,10 @@ OSP_TOKEN_PATH=/tmp/token
 COMMON_DOCKER_ARGS="run --user osg
                         --detach
                         --security-opt apparmor=unconfined
+                        --security-opt seccomp=unconfined
+                        --security-opt systempaths=unconfined
+                        --security-opt no-new-privileges
+                        --net=host
```
Comment (matyasselmeci) [inline tests/test_inside_gha.sh:11]: Switching to host networking is a big change; is that something we're doing just for GHA or are we going to be telling people to do it for their pilots?

Comment (brianhlin) [reply tests/test_inside_gha.sh:11]: We started doing it here https://github.com/opensciencegrid/docs/pull/874

Comment (matyasselmeci) [reply tests/test_inside_gha.sh:11]: Okay, then.
Might want to explain in that document why you are doing it but that's a separate PR.

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**